code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
```

Comment:
The defines are there because that's the way infestation.dm does it. I automatically assume that any previous code is written way way better than I could ever write it.